index.jsx

```diff
@@ -78,6 +78,10 @@ class InlineEdit extends React.Component {
         if(!this.state.editing) {
             return <span className={this.props.className} onClick={this.startEditing}>{this.props.text}</span>
         } else {
+            if (this.props.element === 'textarea') {
+                return <textarea className={this.props.activeClassName} onKeyDown={this.keyDown} onBlur={this.finishEditing} ref="input" defaultValue={this.state.text} onChange={this.textChanged} onReturn={this.finishEditing}></textarea>
```

Comment:
True, I was just think in terms of having a default but we can just do
`const Element = this.props.element || 'input';`